NuRadioMC/simulation/config_default.yaml

```diff
@@ -30,6 +30,9 @@ propagation:
   focusing: False  # if True apply the focusing effect.
   focusing_limit: 2  # the maximum amplification factor of the focusing correction
   n_reflections: 0  # the maximum number of reflections off a reflective layer at the bottom of the ice layer
+  birefringence: False  # if True apply birefringence effects
+  birefringence_propagation: 'analytical'  # can also be 'numerical' - Does not have to be set, default value is 'analytical'
+  birefringence_model: 'A'  # can also be 'B', 'C', 'D', 'E' - Does not have to be set, default value is 'A'
```

Comment:
can you add a link to the documentation, or in general where to find out what A B C etc. mean

Reply:
done